handbook/product/rollout_process.md

```diff
@@ -0,0 +1,44 @@
+# Feature rollout process
+
+Features come in many different sizes and shapes, and the process for introducing new functionality ranges with these differences. For large or significantly impactful changes or changes that simply need a bit more time to bake, it is encouraged that the following rollout process is followed.
+
+## Sourcegraph Cloud
+
+Sourcegraph Cloud is continuously deployed with all new updates to master. We maintain a [releasability contract](../engineering/continuous_releasability.md) and require all new features to be released behind a feature flag to ensure that functionality can be turned off if a problem arises.
+
+### Before merge
+
+- Run hallway tests with internal users
+- Complete a final [design review](design/design_process.md#final-review)
+- Review documentation
+- Review analytics and ensure desired metrics have been added to the feature
+- Confirm feature flag functionality
```

Comment:
Do we need to add a step here where the product team must approve the PR before merging? I worry that the code will be ready, the PR is approved by other developers, but the product team doesn't approve yet. It then becomes unclear when the dev can feel confident merging.

Reply:
Seems like all of this can actually be after merge and deployment if things are feature flagged?